### DISCUSSION TITLE
Can i delete some files?

### DISCUSSION THREAD

**Discussion by gugongerguo** [Chia-Network/chia-blockchain#11151] [Support]
![image](https://user-images.githubusercontent.com/76429627/163112251-986469b4-a581-4ae9-92b3-87e6dec5511b.png)
about 104GB,can i delete this file?

**Comment by gugongerguo** [comment]
another question , why my db file named *_mainnet.sqlite, but the database_path = db/blockchain_v1_CHALLENGE.sqlite in config.yaml?

**Reply by loppefaaret** [reply]
Cause the CHALLENGE will get resolved to whatever "network" you are farming on - if "mainnet", the file will inherit that in the filename - if "testnet10", it would inherit that in its filename
and no, you cannot delete this file, if you run a node (which has a copy of the entire blockchain, and that is what the file is)
what i can recommend, is that you upgrade to 1.3.X, and look into upgrading your database from "v1" to "v2" - this will save you about 45% space for that database file.
Read more about it here:
https://github.com/Chia-Network/chia-blockchain/wiki/CLI-Commands-Reference#db
